index.bs

```diff
@@ -1092,15 +1092,19 @@ To <dfn>verify a certificate hash</dfn>, do the following:
 </div>
 
 The <dfn>custom certificate requirements</dfn> are as follows: the certificate
-MUST be an X.509v3 certificate as defined in [[!RFC5280]], the current time
-MUST be within the validity period of the certificate as defined in Section
-4.1.2.5 of [[!RFC5280]] and the total length of the validity period MUST NOT
-exceed two weeks.
-
-Issue: Reconsider the time period above.  We want it to be sufficiently large
-that applications using this for ephemeral certificates can do so without
-having to fight the clock skew, but small enough to discourage long-term use
-without key rotation.
+MUST be an X.509v3 certificate as defined in [[!RFC5280]], the key used in the
+Subject Public Key field MUST be one of the [=allowed public key algorithms=],
+the current time MUST be within the validity period of the certificate as
+defined in Section 4.1.2.5 of [[!RFC5280]] and the total length of the validity
+period MUST NOT exceed two weeks.  The user agent MAY impose additional
```

Comment:
Does the time period need to be specified in a way that's robust against leap days/seconds/etc? Perhaps specifying 14 days, and referring to [6.3.2 of the Baseline Requirements](https://cabforum.org/wp-content/uploads/CA-Browser-Forum-BR-1.8.0.pdf#page=70)' definition of a day, and recommendations for issuance?

Reply:
Also: two weeks is probably a reasonable compromise, but it would be helpful to understand how you landed on it. The discussion in the section below is light on detail...

Reply:
I believe two weeks, 14 days, and 14 * 24 * 3600 seconds are all equivalent. None of those measures are affected by leap days. Leap seconds are horrible, but all timestamps in certificates (and the web platform, and most of software in general) uses UTC, not TAI. That means leap seconds aren't counted in these sorts of measures. They just silently repeat/smear relative to physical time.

Reply:
Added a comment on the rationale behind two weeks.